charts/pulsar/templates/proxy/proxy-statefulset.yaml

```diff
@@ -241,7 +241,7 @@ spec:
         - configMapRef:
             name: "{{ template "pulsar.fullname" . }}-{{ .Values.proxy.component }}"
         env:
-{{- with .Values.broker.extraEnv}}
+{{- with .Values.pulsar.extraEnv}}
```

Comment:
`proxy`?

Reply:
Yes......my mistake....

Reply:
fixed in https://github.com/streamnative/charts/pull/659/commits/be74a1821f38523906f906d3e5f07746fe0f9ec1